### DISCUSSION TITLE
与DDNS冲突，启动后DDNS无法更新IP

### DISCUSSION THREAD

**Discussion by liumingan** [vernesong/OpenClash#2721] [Q&A]
同标题

**Comment by openips** [comment]
没遇到过这样的问题 国内国外的ddns都在用 只不过需要把国外ddns检测网址配置为直连

**Comment by vernesong** [comment]
自定义规则里面有预设

**Reply by liumingan** [reply]
不启动openclash的话一切正常，启动后DDNS就报以下错误,自定义规则已经把dynv6.com设置为直连：
requested host name 'dynv6.com'.
To connect to dynv6.com insecurely, use `--no-check-certificate'.
 225023  WARN : Transfer failed - retry 302/0 in 60 seconds
 225123       : #> /usr/bin/wget --hsts-file=/tmp/.wget-hsts -nv -t 1 -O /var/run/ddns/myddns_ipv6.dat -o /var/run/ddns/myddns_ipv6.err --bind-address=2408:824e:530b:35b0::1 --no-proxy 'http://dynv6.com/api/update?hostname=myhome.dynv6.net&token=***PW***&ipv6=2408:824e:530b:35b0:0000:0000:0000:0001'
 225124 ERROR : GNU Wget Error: '5'
 225124       : ERROR: cannot verify dynv6.com's certificate, issued by 'CN=OpenWrt,O=OpenWrt00d3f24c,L=Unknown,ST=Somewhere,C=ZZ':
  Self-signed certificate encountered.
ERROR: no certificate subject alternative name matches

requested host name 'dynv6.com'.
To connect to dynv6.com insecurely, use `--no-check-certificate'.
 225124  WARN : Transfer failed - retry 303/0 in 60 seconds
 225224       : #> /usr/bin/wget --hsts-file=/tmp/.wget-hsts -nv -t 1 -O /var/run/ddns/myddns_ipv6.dat -o /var/run/ddns/myddns_ipv6.err --bind-address=2408:824e:530b:35b0::1 --no-proxy 'http://dynv6.com/api/update?hostname=myhome.dynv6.net&token=***PW***&ipv6=2408:824e:530b:35b0:0000:0000:0000:0001'
 225224 ERROR : GNU Wget Error: '5'
 225224       : ERROR: cannot verify dynv6.com's certificate, issued by 'CN=OpenWrt,O=OpenWrt00d3f24c,L=Unknown,ST=Somewhere,C=ZZ':
  Self-signed certificate encountered.
ERROR: no certificate subject alternative name matches

requested host name 'dynv6.com'.
To connect to dynv6.com insecurely, use `--no-check-certificate'.
 225224  WARN : Transfer failed - retry 304/0 in 60 seconds

**Reply by liumingan** [reply]
官方开发版openwrt

**Reply by liumingan** [reply]
奇怪的是同样运行官方op的下级路由器DDNS更新正常，只是运行openclash 的路由器上出现这个问题

**Reply by millionfor** [reply]
我也遇到了这个问题 有解决吗
